buildpack.toml

```diff
@@ -62,6 +62,21 @@ api = "0.5"
     uri = "https://buildpacks.cloudfoundry.org/dependencies/go/go_1.17_linux_x64_cflinuxfs3_668718ba.tgz"
     version = "1.17"
 
+  [[metadata.dependency-constraints]]
+    constraint = "1.15.*"
+    id = "go"
+    patches = 2
```

Comment:
Should we not add this constraint and remove the 1.15.x line as 1.17.x is out?

Reply:
Good question. I'm not sure. I only translated what was currently in the `buildpack.toml` in terms of versions into the dependency constraint format. If we remove 1.15, do we need a EOL notice // should we announce somehow?

Reply:
I'm going to merge as-is for now. We can decide re: 1.15 later.

Reply:
Sounds good to me. Will review.